### DISCUSSION TITLE
Is it possible to modify output from config.py ?

### DISCUSSION THREAD

**Discussion by justinesmithies** [qtile/qtile#3319] [Q&A] [answered]
Is there a way that I could do away with Kanshi and have a hook do all the work of commands like this sample line from my Kanshi config ?

`output eDP-1 position 0,0 mode 1920x1080`

**Comment by m-col** [comment, marked as answer]
I'm afraid it's a bit out of scope for Qtile (like `xrandr`) but if you don't need a daemon monitoring for output changes then you can use a one-shot utility to configure things at startup and exit. Check out `wlr-randr`, for instance.
